tests/ChatClient.hs

```diff
@@ -287,7 +287,10 @@ readTerminalOutput t termQ = do
 withTmpFiles :: IO () -> IO ()
 withTmpFiles =
   bracket_
-    (createDirectoryIfMissing False "tests/tmp")
+    ( do
+        createDirectoryIfMissing False "tests/tmp"
+        createDirectoryIfMissing False "tests/tmp/tmp" -- for XFTP temporary files
```

Comment:
this change should not be needed, it should create this directory. If it doesn't, better to fix it.

Reply:
it's in tests

Reply:
It's set in SetTempFolder API, and then passed to agent only on chat start. Folder is also created in SetTempFolder. So in tests it would require stopping chat, setting folder, and starting chat in each test. There is such test - testXFTPWithRelativePaths. To not clutter all other tests better to just have this directory prepared and avoid this workaround.

Reply:
Previously it was just configured to tests/tmp, which is a tests temp folder and this sometimes conflicts with test logic, for example when database is deleted and it deletes whole tests temp folder instead of only necessary folders, so some tests already configured this additional nested folder as temporary and created it at the start of test.